spec/unit/processors/os_spec.rb

```diff
@@ -221,6 +221,7 @@ def load(procs)
       before :each do
         proc = stubs 'proc'
         proc.stubs(:Name).returns("Intel(R)    Celeron(R)   processor")
+        proc.stubs(:ole_respond_to?).with(:NumberOfLogicalProcessors).returns(false)
```

Comment:
I don't think we need an explicit `never` expectation since it's a stub object and it will raise if any other methods (besides `Name`) are called on it.